recipes-security/optee-imx/optee-os_3.7.0.imx.bb

```diff
@@ -22,19 +22,18 @@ S = "${WORKDIR}/git"
 B = "${WORKDIR}/build.${PLATFORM_FLAVOR}"
 
 # The platform flavor corresponds to the Yocto machine without the leading 'i'.
-PLATFORM_FLAVOR                 = "${@d.getVar('MACHINE')[1:]}"
-PLATFORM_FLAVOR_imx6qpdlsolox   = "mx6qsabresd"
-PLATFORM_FLAVOR_imx6ul7d        = "mx6ulevk"
-PLATFORM_FLAVOR_imx6ull14x14evk = "mx6ullevk"
-PLATFORM_FLAVOR_imx6ull9x9evk   = "mx6ullevk"
-PLATFORM_FLAVOR_imx6ulz14x14evk = "mx6ulzevk"
-PLATFORM_FLAVOR_mx8mm   	= "mx8mmevk"
-PLATFORM_FLAVOR_mx8mn   	= "mx8mnevk"
-PLATFORM_FLAVOR_mx8qxp 		= "mx8qxpmek"
-PLATFORM_FLAVOR_mx8mp   	= "mx8mpevk"
-PLATFORM_FLAVOR_mx8dx   	= "mx8dxmek"
-PLATFORM_FLAVOR_mx8dxl  	= "mx8dxlevk"
-PLATFORM_FLAVOR_mx8phantomdxl 	= "mx8qxpmek"
+PLATFORM_FLAVOR                   = "${@d.getVar('MACHINE')[1:]}"
+PLATFORM_FLAVOR_imx6qpdlsolox     = "mx6qsabresd"
+PLATFORM_FLAVOR_imx6ul7d          = "mx6ulevk"
+PLATFORM_FLAVOR_imx6ull14x14evk   = "mx6ullevk"
+PLATFORM_FLAVOR_imx6ull9x9evk     = "mx6ullevk"
+PLATFORM_FLAVOR_imx6ulz-14x14-evk = "mx6ulzevk"
+PLATFORM_FLAVOR_mx8mm             = "mx8mmevk"
+PLATFORM_FLAVOR_mx8mn             = "mx8mnevk"
+PLATFORM_FLAVOR_mx8qxp            = "mx8qxpmek"
+PLATFORM_FLAVOR_mx8mp             = "mx8mpevk"
+PLATFORM_FLAVOR_mx8dx             = "mx8dxmek"
+PLATFORM_FLAVOR_mx8dxl            = "mx8dxlevk"
```

Comment:
Unnecessary re-formatting changes here. Why all tabs are substituted with spaces?

Reply:
Well, if you look at the before and after above perhaps you can see why I use spaces consistently? ;-) Mixed tabs and spaces (fixed width) don't always show well depending on the user's settings, while spaces always work.